cvat-sdk/gen/postprocess.py

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env python
 
-# Copyright (C) 2022 Intel Corporation
+# Copyright (C) 2022 CVAT.ai Corporation
```

Comment:
I think we're supposed to add the new copyright line on top, not replace the old one.

Reply:
It was a wrong one.